core/sql/regress/newregr/mvs/BIG/EXPECTEDMV215C

```diff
@@ -20,21 +20,6 @@
 -- @@@ END COPYRIGHT @@@
 -- @@@ START COPYRIGHT @@@
```

Comment:
It is still there. Will remove the erroneous @@@ START COPYRIGHT sign. The newregr directory has some missing copyrights too. Will fix on next pass.